internal/view/svc.go

```diff
@@ -60,6 +60,9 @@ func (s *Service) showPods(a *App, _ ui.Tabular, gvr, path string) {
 		return
 	}
 
+	if svc.Spec.Type == "ExternalName" {
```

Comment:
Please use the builtin constant v1.ServiceTypeExternalName. IMHO no point in make a call that returns nothing. Also let's put up a flash message to the effect that this is an external svc and hence yeilds no pods. What do you think?